toxcore/Messenger.h

```diff
@@ -200,15 +200,11 @@ typedef void m_friend_lossy_packet_cb(Messenger *m, uint32_t friend_number, uint
                                       size_t length, void *user_data);
 typedef void m_friend_lossless_packet_cb(Messenger *m, uint32_t friend_number, uint8_t packet_id, const uint8_t *data,
         size_t length, void *user_data);
-typedef void m_friend_connectionstatuschange_internal_cb(Messenger *m, uint32_t friend_number,
-        bool is_online, void *user_data);
 typedef void m_conference_invite_cb(Messenger *m, uint32_t friend_number, const uint8_t *cookie, uint16_t length,
                                     void *user_data);
 typedef void m_group_invite_cb(const Messenger *m, uint32_t friend_number, const uint8_t *invite_data, size_t length,
                                const uint8_t *group_name, size_t group_name_length, void *user_data);
-typedef void m_msi_packet_cb(Messenger *m, uint32_t friend_number, const uint8_t *data, uint16_t length,
-                             void *user_data);
-typedef int m_lossy_rtp_packet_cb(Messenger *m, uint32_t friend_number, const uint8_t *data, uint16_t length, void *object);
+typedef int m_lossy_rtp_packet_cb(Messenger *m, uint32_t friend_number, const uint8_t *data, uint16_t len, void *object);
```

Comment:
len? -> length

Reply:
good catch.
"m_lossy_rtp_packet_cb" is actually a leftover that should be deleted. same as "RTP_Packet_Handler"

Reply:
do we have some sort of "identify unused functions and struct defs" scanner?

Reply:
removed